src/loader.c

```diff
@@ -201,6 +201,12 @@ bool parse_database(void *base, const char *name, const char *connstr)
 	cv.value_p = &pool_mode;
 	cv.extra = (const void *)pool_mode_map;
 
+	/* pgbouncer pool is set by admin_setup() */
+	if (strcmp(name, "pgbouncer") == 0) {
+		log_error("skipping database %s because it is set internally", name);
+		return true;
```

Comment:
I'm surprised by the error handling in this function. It seems true is returned even in error cases. But maybe we should at least use `goto fail` like other cases?

The error message isn't really strong enough for an error. It should be something like "database name %s is reserved".

Reply:
`goto` was the first thing I thought but decided not to use it because I would have to add an `if` to handle `free`. The other option is to move the code a few lines below (after `strdup`) but I decided that it doesn't worth it.

I will change the wording as you suggested.